firmware/src/widgets/weatherwidget/WeatherWidget.h

```diff
@@ -57,17 +56,15 @@ class WeatherWidget : public Widget {
     #define WEATHER_LOCATION WEATHER_LOCAION
 #endif
 
-    const String weatherLocation = WEATHER_LOCATION;
+    std::string m_weatherLocation = WEATHER_LOCATION;
+
 #ifdef WEATHER_UNITS_METRIC
-    const String weatherUnits = "metric";
+    int m_weatherUnits = 0;
 #else
-    const String weatherUnits = "us";
+    int m_weatherUnits = 1;
```

Comment:
This should probably be an enum?

Reply:
An enum would be nicer, but enum doesn't work with ConfigManager at the moment.
While registering a config value, we have to pass the ConfigManager a reference to a variable (&int in this case).

Maybe I can cast the enum to an int and pass that as a reference? Will the changes to this referenced (cast) int still be reflected in the original enum variable?